cmake/options.cmake

```diff
@@ -0,0 +1,26 @@
+# Options
```

Comment:
Should we have another option to build with-fPIC for static library build, similar as --with-pic configure option?

Reply:
Yes, I can add it. Should the default be on or off?

Reply:
ON for static lib only.